website/docs/plugin/framework/migrating/attributes-blocks/validators-predefined.mdx

```diff
@@ -13,6 +13,20 @@ the Framework documentation for details. Refer to the
 [Attributes - Custom Validators](/plugin/framework/migrating/attributes-blocks/validators-custom) page in this guide to learn how to
 implement custom validators.
 
+The following table describes the mapping between predefined validators in SDKv2 and the Framework. You should use an
+attribute validator for [per-attribute validation](/plugin/framework/validation#attribute-validation), or a
+[data source validator](/plugin/framework/data-sources/validate-configuration),
+[provider validator](plugin/framework/providers/validate-configuration) or
+[resource validator](/plugin/framework/resources/validate-configuration) for declaring validation at the level of the
+data source, provider or resource, respectively.
+
+| SDK Attribute Field | Framework Attribute Validator   | Framework Data Source, Provider and Resource Validators                                                                                                                             |
+|---------------------|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AtLeastOneOf        | {TYPE}validator.AtLeastOneOf()  | [datasourcevalidator.AtLeastOneOf()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-validators/datasourcevalidator#AtLeastOneOf), [providervalidator.AtLeastOneOf()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-validators/providervalidator#AtLeastOneOf), [resourcevalidator.AtLeastOneOf()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework-validators/resourcevalidator#AtLeastOneOf)         |
```

Comment:
Would some html be useful here to have a list for the cases where there are several translation available: https://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table

Reply:
I'm not sure how well our Markdown rendering engine for the website plays with intermixed HTML/Markdown syntax. Another option here could be having separate table columns for data source, provider, and resource to prevent the listing in general.

Reply:
Have split into 3 columns for data source, provider and resource.